src/20_Components/amp-bind.html

```diff
@@ -101,27 +101,29 @@
    set when a state change is triggered (see below).
 
 -->
-<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+<div class="p1">
+  <p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+</div>
 
 <!-- ## Binding Styles -->
 <!--
   Styling can be changed by applying css classes. It will override all the element's style classes. `allAnimals` is the `id` of the &lt;amp-state&gt; component above.
 -->
-
-<p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
-
+<div class="p1">
```

Comment:
Margin unfortunately doesn't work, snapshot attached.
<img width="1280" alt="screen shot 2017-03-02 at 3 27 48 pm" src="https://cloud.githubusercontent.com/assets/820891/23513697/db9e4b46-ff5c-11e6-934d-0606a62266ee.png">
Adding a padding to the p itelf also doesn't work as it enlarges the background. I tonly works having a parent element with a padding